library/webpack-plugins/websocket-communication-plugin.js

```diff
@@ -80,8 +80,14 @@ function findFreePort(retries = 2) {
 }
 
 function startWebSocketServer(port) {
-  console.log(`WebSocket opened on port ${port}`)
   const wss = new ws.Server({ port })
+  wss.on('connection', ws => {
+    ws.on('error', err => {
```

Comment:
Did you check that the error was caught here?

Reply:
Yep

Reply:
My environment had 100% failure which was fixed by this commit.